rir/src/compiler/translations/rir_2_pir/stack_machine.cpp

```
@@ -413,11 +434,9 @@ void StackMachine::runCurrentBC(const Rir2Pir& rir2pir, Builder& insert) {
     case Opcode::endcontext_:
     case Opcode::ldddvar_:
     case Opcode::int3_:
-        std::cerr << "Cannot compile Function. Unsupported bc\n";
-        bc.print();
-        assert(false);
-        break;
+        return false;
```

Maybe move the currently unused ones and the ones from pir here as well?

---

I think it's still ok to fail hard on the ones we do not expect to see